samples/openthread/common/overlay-ot-defaults.conf

```diff
@@ -24,26 +24,26 @@ CONFIG_NET_LOG=y
 CONFIG_NET_STATISTICS=y
 
 # Disable certain parts of Zephyr IPv6 stack
-#CONFIG_NET_CONFIG_SETTINGS=y
 CONFIG_NET_IPV6_NBR_CACHE=n
 CONFIG_NET_IPV6_MLD=n
 
 # Stack sizes configuration
-CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
-CONFIG_NET_TX_STACK_SIZE=2048
-CONFIG_NET_RX_STACK_SIZE=5120
-CONFIG_SHELL_STACK_SIZE=3072
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1024
```

Comment:
All these new stack sizes are Zephyr defaults, perhaps we could just remove them from this file?

Reply:
That's what I first did, but then I brought them back in order to make it easier to modify them in the future if needed. What about leaving them here but commented. I'm fine with removing them anyway.

Reply:
I don't find keeping these configs a big issue if you prefer it this way. Let's keep them as they are, uncommented.